neurogym/envs/contextdecisionmaking.py

```diff
@@ -15,21 +15,22 @@ class ContextDecisionMaking(ngym.TrialEnv):
     modalities) and needs to make a perceptual decision based on one while
     ignoring the other. The task can operate in two modes:
     1. Implicit context: The relevant modality is fixed and must be learned (
-       it is not explicitly signaled). Uses ring representation.
+       it is not explicitly signaled).
     2. Explicit context: The agent simultaneously receives stimulus inputs
        from two modalities, and the relevant modality is explicitly indicated
-       by a context signal. Uses direct value representation.
+       by a context signal.
+    Both modes use ring representation for encoding stimulus inputs and choices.
 
     Args:
         dt: int, timestep of the environment.
         use_expl_context: bool, if True, the context is explicit (signaled) and changes per trial.
-            If true, direct value representation is used, otherwise ring representation.
         impl_context_modality: int, which fixed implicit modality to use if `use_expl_context`
             is False (0 or 1).
-        impl_dim_ring: int, number of choices when using implicit representation.
+        dim_ring: int, number of choices in both modalities.
```

Comment:
I think this statement is more a "historic" thing to explain the change. Not needed going forward, as it is documented above as well.

```suggestion
        dim_ring: int, number of choices.
```